pkg/loadbalancers/l4netlb.go

```
@@ -100,118 +112,115 @@ func (l4netlb *L4NetLB) createKey(name string) (*meta.Key, error) {
 // been created. It is health check, firewall rules, backend service and forwarding rule.
```

I don't like this design here the error is hidden and it is easy to forget about error handling from function.